src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -1601,6 +1627,26 @@ namespace ZoneTempPredictorCorrector {
 							}
 						}
 
+						// added Jan, 2017 - Xuan Luo
+						// read adaptive comfort model and calculate adaptive thermal comfort setpoint
+						if ( TempControlledZone( TempControlledZoneNum ).OperativeTempControl ) {
+							if ( NumAlphas == 4 ) {
+								int adaptiveComfortModelTypeIndex = FindItem( cAlphaArgs( 4 ), AdaptiveComfortModelTypes, AdaptiveComfortModelTypes.isize() );
+								if ( adaptiveComfortModelTypeIndex != ADAP_NONE ) {
```

Comment:
If adaptiveComfortModelTypeIndex == ADAP_NONE, do you provide additional message? For example, if it is wrong spelling, do you set ErrorsFound = true?